### PR TITLE
Set sshd keys

### PR DESCRIPTION
**Pull request by jerome-jutteau**

### CODE REVIEW

File: github_actions/deploy_cluster/action.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ inputs:
   image_id:
     description: 'Image ID used for all VMs'
     required: false
-    default: "ami-d76e520a"
+    default: "ami-9b41b457"
```
Comment (outscale-vbr) [inline github_actions/deploy_cluster/action.yml:92]: we are using ci on us-east-2 ?

File: github_actions/deploy_cluster/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ See [action.yml](action.yml)
 | `worker_volume_type`        | Volume type of worker instance                                                               | `false`  | `"standard"`                            |
 | `worker_volume_size`        | Volume size of worker instance                                                               | `false`  | `15`                                    |
 | `worker_volume_iops`        | Volume iops of worker instance                                                               | `false`  | `1500`                                  |
-| `image_id`                  | Image ID used for all VMs                                                                    | `false`  | `ami-d76e520a` (available in eu-west-2) |
+| `image_id`                  | Image ID used for all VMs                                                                    | `false`  | `ami-9b41b457` (available in eu-west-2) |
```
Comment (outscale-vbr) [inline github_actions/deploy_cluster/README.md:33]: we are using ci on us-east-2

File: docs/deploy.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Prerequisite
 
 - [Terraform](https://www.terraform.io/downloads) (>= 0.14)
-- [RKE](https://rancher.com/docs/rke/latest/en/installation/) (>= v1.4.1)
```
Comment (outscale-vbr) [inline docs/deploy.md:4]: Why you remove rke ?

**Comment by outscale-vbr** [comment]
@jerome-jutteau wat is the purpose of adding sshd keys ?

File: docs/deploy.md (hunk 3 above)
Comment (jerome-jutteau) [reply docs/deploy.md:4]: Not mandatory anymore as rke binary is downloaded

**Comment by jerome-jutteau** [comment]
> @jerome-jutteau what is the purpose of adding sshd keys ?

This is a legitimate request from issue #30: we can easily avoid "accept-new" situation / potential MITM attacks.

File: github_actions/deploy_cluster/README.md (hunk 2 above)
Comment (jerome-jutteau) [reply github_actions/deploy_cluster/README.md]: fixed

File: github_actions/deploy_cluster/action.yml (hunk 1 above)
Comment (jerome-jutteau) [reply github_actions/deploy_cluster/action.yml:92]: yes

**Comment by jerome-jutteau** [comment]
Need further public-cloud debug which seems related to the CI, works on my side.

**Pull request merged**